apps/explorer/src/pages/transaction-result/programmable-transaction-view/Inputs.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiJsonValue, toB64 } from '@mysten/sui.js';
+
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    inputs: SuiJsonValue[];
```

Comment:
Can probably use the new refined type here.

Reply:
Changed to reflect new changes from sdk